nitro-protocol/contracts/ForceMove.sol

```diff
@@ -142,7 +142,8 @@ contract ForceMove is IForceMove, StatusManager {
         SignedVariablePart[] memory proof,
         SignedVariablePart memory candidate
     ) internal returns (bytes32 channelId) {
-        _requireCorrectChainId(fixedPart.chainId);
+        // TODO: The FEVM doesn't seem to support the CHAINID opcode.
+        // _requireCorrectChainId(fixedPart.chainId);
```

Comment:
It just so happens we discussed the idea of not doing this check (not even having the chain id in the fixed part) as part of RFC 16 yesterday 🤷 .